workers/gunicorn_worker.py

```diff
@@ -28,7 +30,7 @@ def __init__(self, name, app, worker, feature_flag):
         if self.feature_flag:
             self.logger.debug("starting {} thread".format(self.name))
             p = Process(target=self.worker.start)
-            p = p.start()
+            p.start()
 
-    def __call__(environ, start_response):
-        return self.app(environ, start_response)
+    def __call__(self, environ, start_response):
```

Comment:
This mistake implies that this function has never been called in your environments.